libs/output-mapping/tests/Writer/Workspace/WriterWorkspaceTest.php

```diff
@@ -112,6 +115,130 @@ public function testSnowflakeTableOutputMapping(): void
         ]);
     }
 
+    protected function prepareWorkspaceWithTablesWithTimestampColumn(
```

Comment:
Tenhle test tu je nějak navíc a nepatří k tomuhle issue ne?

Reply:
Ojoj, to sem opravdu nepatří, mažu